### PR TITLE
Add Skills Necklace map.

### PR DESCRIPTION
**Pull request by Blubmin**
Adds an interactive map to replace the Skills Necklace selection menu. Only includes bare-bones features since all the label-related ones that Xeric and Obelisks have were causing a lot of crowding when enabled.

image

### CODE REVIEW

**Comment by MJHylkema** [comment]
@Blubmin great job! I'll take a closer look tomorrow and merge 👍🏼

**Comment by Blubmin** [comment]
@MJHylkema bump!

File: src/main/java/com/mjhylkema/TeleportMaps/components/adventureLog/SkillsNecklaceMap.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	private static final int SCRIPT_TRIGGER_KEY = 1437;
+	private static final String LABEL_NAME_PATTERN = "<col=735a28>(.+)</col>: (<col=5f5f5f>)?(.+)";
+	private static final String TRAVEL_ACTION = "Travel";
+	private static final String EXAMINE_ACTION = "Examine";
+	private static final int ADVENTURE_LOG_CONTAINER_BACKGROUND = 0;
+	private static final int ADVENTURE_LOG_CONTAINER_TITLE = 1;
+
+	// There has got to be a better way to do this. All other jewelry seems to use the chat window.
+	// But they all say this same phrase. Lucky that skills necklace uses the Adventure Log.
+	private static final String MENU_TITLE = "Where would you like to teleport to?";
```
Comment (MJHylkema) [inline src/main/java/com/mjhylkema/TeleportMaps/components/adventureLog/SkillsNecklaceMap.java:44]: Beyond tracking inventory / item interactions, I'm not really sure of a better way to do this. Happy with this for now 👍🏼

**Review by MJHylkema — APPROVED** [review]
Made a couple of minor changes, but overall looks good. Thanks for the feature!

**Pull request merged**